sdk/java-sdk-protobuf-testkit/src/main/java/kalix/javasdk/testkit/junit/KalixTestKitResource.java

```diff
@@ -41,14 +41,14 @@
  * <p>Example:
  *
  * <pre>
- * import kalix.javasdk.testkit.junit.KalixTestKitResource;
+ * import kalix.javasdk.testkit.junit.jupiter.KalixTestKitExtension;
  *
  * public class MyKalixIntegrationTest {
  *
  *   private static final Kalix MY_KALIX = new Kalix(); // with registered services
  *
  *   &#64;ClassRule
- *   public static final KalixTestKitResource testKit = new KalixTestKitResource(MY_KALIX);
+ *   public static final KalixTestKitExtension testKit = new KalixTestKitExtension(MY_KALIX);
```

Comment:
This looks wrong, this is the `KalixTestKitResource` source file.

Reply:
yep, too many find-and-replace